### PR TITLE
Feature/gcb 280 new blogpost design

### PR DESCRIPTION
**Pull request by michallaskowski**
Make sure these boxes are checked before submitting your Pull Request - thank you!

- [ ] [SwiftLint](https://github.com/Realm/SwiftLint) is installed on your system
- [ ] The project does not generate any warnings or errors
- [ ] The tests are passing
- [ ] Branch name is in line with [GitFlow](http://jeffkreeftmeijer.com/2010/why-arent-you-using-git-flow/)

I left option for headers (because this TableView controlling stack is nice), but it could be removed too. Keep in mind this just a content view of the widget, the header and subheader will be added in another PR, when they are ready.

### CODE REVIEW

**Comment by krris** [comment]
There should be spacing between first element and the top of the widget.

**Comment by michallaskowski** [comment]
Yes, but I want to add the spacing when we have a template view for a widget. Because it may affect how it will be done (like we discussed, this template view maybe will have parametrizable margins).

File: GrandCentralBoard/Widgets/WebsiteAnalytics/View/HeaderCell.swift [hunk 1]
```diff
@@ -17,26 +17,29 @@ protocol ViewDequeuing {
...
+    init(cellNib: UINib, cellIdentifier: String, headerNib: UINib?, headerIdentifier: String?, tableView: UITableView) {
         self.cellNib = cellNib
         self.cellIdentifier = cellIdentifier
         self.headerNib = headerNib
         self.headerIdentifier = headerIdentifier
         self.tableView = tableView
 
         tableView.registerNib(cellNib, forCellReuseIdentifier: cellIdentifier)
-        tableView.registerNib(headerNib, forHeaderFooterViewReuseIdentifier: headerIdentifier)
+        if let headerNib = headerNib, let headerIdentifier = headerIdentifier {
```
Comment (krris) [inline GrandCentralBoard/Widgets/WebsiteAnalytics/View/HeaderCell.swift]: There is a warning ⚠️   `Repeated `let` statements in conditional binding cascade should be avoided. (conditional_binding_cascade)`

**Comment by krris** [comment]
CR is finished. Please rebase.

**Comment by michallaskowski** [comment]
Closing, will reopen the same one but for a different ticket.

**Pull request closed without merging**